src/App.js

```diff
@@ -3,6 +3,7 @@ import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
 import Navigation from "./components/Navigation";
 import Home from "./components/Home";
```

Comment:
The Components foulder will also start with a capital letter, same as individual Components

Reply:
I disagree
found an article about structure:
https://hackernoon.com/structuring-projects-and-naming-components-in-react-1261b6e18d76